web/setups/sync.php

```diff
@@ -0,0 +1,22 @@
+<?php
+/**
+ * Copyright (c) 2017 University of Illinois, NCSA.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the
+ * University of Illinois/NCSA Open Source License
+ * which accompanies this distribution, and is available at
+ * http://opensource.ncsa.illinois.edu/license.html
+ */
+
+// sync
+
+$outcome = NULL;
+$status = NULL;
+
+$flagfile = fopen("syncflag.txt","r");
+$want_id = fgets ($flagfile);
+fclose($flagfile);
+
+exec('../../scripts/load.bety.sh', $outcome, $status);
```

Comment:
This doesn't seem to be sufficient to sync. 

First, whenever I run the sync scripts I have to put MYSITE=1 before the script call (or use flags to specify what my host id is)

Second, my sync script always starts with a dump.bety.sh, otherwise the sync is just one way

Third, currently when running load.bety.sh I have to loop over every remote site explicitly, setting REMOTESITE=<n>, for each.

FWIW, in more advanced applications, I think it would be great to allow the user to interactively choose which sites they want to sync with, but with "all" being the default. It would also be good to eventually give the user more control over what they push out during dump.bety.sh (in terms of tables and access levels)